deploy/examples/cluster.yaml

```
@@ -222,6 +222,8 @@ spec:
 # nodes below will be used as storage resources.  Each node's 'name' field should match their 'kubernetes.io/hostname' label.
     # nodes:
     #   - name: "172.17.4.201"
+    #     config:
+    #       metadataDevice: "" # metadataDevice can be specified using full udev path
```

Actually there is already a config section for this node below on line 233. How about if you move this line into that existing section?

For the comment, let's be clear that device name can also be used
```suggestion
    #       metadataDevice: "" # metadataDevice can be specified using device name or full udev path
```

---

@travisn that section of the config is specific for node device, but this is for a particular node.
And right now we only query metadatadevice config key for node, right?

---

Yup, right.